python/knlc/kn_strategy_sims_MI.py

```diff
@@ -2228,6 +2243,7 @@ def get_detection_later(p_, p_later, start_time, delay, start_later):
                     "Region_coverage_deg_deep": area_deg_all_deep,
                     "Detprob1": prob1_all,
                     "Detprob2": prob2_all,}).to_csv(f,  index=False)
+    os.chmod(f, 0o0777)
```

Comment:
If f is a TextIOWrapper class or similar, won't work. Should be the name of the file instead (like a string)

Reply:
Resolved to `strategy_file` variable, which is a string